### PR TITLE
chore: improve Travis CI config

### PR DESCRIPTION
**Pull request by DanielRuf**

### CODE REVIEW

**Comment by DanielRuf** [comment]
CI should succeed now.

**Comment by DanielRuf** [comment]
https://travis-ci.org/chinchang/web-maker/jobs/396308073#L544

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+  - npm i -g npm@latest
   - npm install -g eslint
   - npm install -g babel-eslint
   - npm install -g eslint-plugin-react
   - npm install -g eslint-plugin-mocha
   - npm install -g eslint-config-synacor
 script:
   - npm run lint
+  - npm ci
+  - npm run test
```
[the remark this answers is not in the document]
Comment (DanielRuf) [reply .travis.yml]: Either we fix these (should not be hard ;-) ) or we comment out the last line. Which option should I choose? I'll do the changes for you. There is no reason for "but" ;-)

And this way any contributor can provide new tests and does not have to enable them in the CI config.

[the remark this answers is not in the document]
Comment (chinchang) [reply .travis.yml]: Awesome! Then, let's fix them.

[the remark this answers is not in the document]
Comment (DanielRuf) [reply .travis.yml]: I guess I fixed it, the tests are currently running on CI.

**Pull request merged**